packages/flutter/test/material/app_bar_test.dart

```diff
@@ -1672,7 +1672,7 @@ void main() {
       ),
     );
     await tester.tap(find.byKey(key));
-    expect(painter, paints..save()..translate()..save()..translate()..circle(x: 24.0, y: 28.0));
+    expect(painter, paints..save()..translate()..save()..translate()..circle(x: 24.0, y: 24.0));
```

Comment:
It's odd that this used to be `y: 28`. I don't know why it was that way, but 24 makes more sense.

Reply:
I think the 28 is because the `AppbBar` height is 56 pixels, and the `crossAxisAlignment` was set to `CrossAxisAlignment.stretch` previously, so the action takes all the space of the height. After removing the `crossAxisAlignment` setting, the action button now is 48 pixels in height and has 24 as result.

Reply:
I see, thanks for the explanation.